### ISSUE TITLE
Html Parsing Bug

### ISSUE THREAD

**Issue by MatthewGilbert** [danielfrg/pelican-ipynb#27]
I'm attempting to convert an ipython notebook to html but there seems to be an issue parsing the json image component of a notebook.

![selection_014](https://cloud.githubusercontent.com/assets/2401026/7791205/0e4be298-026a-11e5-82a3-4ec01fecd5e9.png)

It appears that the string is being improperly escaped

![selection_015](https://cloud.githubusercontent.com/assets/2401026/7791239/ea29cffa-026a-11e5-99c9-4b1b37786ca1.png)

I believe it has something to do with when the text is parsed using BeautifulSoup but have been unable to pin down exactly what the issue is. I have a repo which has a single blog post where the issue is occuring.

https://github.com/MatthewGilbert/blogbug

**Comment by linhua55** [comment]
Which pelican theme do you use?
I tested your ipynb file with my selected theme,and there is no problem.
http://wen.lhwebw.tk/terst2.html

**Comment by MatthewGilbert** [comment]
Hmm, yes I was having some issues reproducing the bug but the repo I have up seems to reliable (I believe) reproduce the error. I'm using the default Pelican theme with pelican version 3.5.0 and Python 2.7.9

The debugging info from the run is included

```
make DEBUG=1 html
pelican /home/matthew/Projects/blogbug/content -o /home/matthew/Projects/blogbug/output -s /home/matthew/Projects/blogbug/pelicanconf.py  -D
DEBUG: Adding current directory to system path
DEBUG: Temporarily adding PLUGIN_PATHS to system path
DEBUG: Loading plugin `pelican-ipynb`
DEBUG: Registering plugin `pelican-ipynb`
DEBUG: Restoring system path
DEBUG: Cannot load cache /home/matthew/Projects/blogbug/cache/ArticlesGenerator-Readers (this is normal on first run). Proceeding with empty cache.
  | [Errno 2] No such file or directory: u'/home/matthew/Projects/blogbug/cache/ArticlesGenerator-Readers'
DEBUG: Template list: [u'!simple/archives.html', u'!simple/article.html', u'!simple/author.html', u'!simple/authors.html', u'!simple/base.html', u'!simple/categories.html', u'!simple/category.html', u'!simple/gosquared.html', u'!simple/index.html', u'!simple/page.html', u'!simple/pagination.html', u'!simple/period_archives.html', u'!simple/tag.html', u'!simple/tags.html', u'!simple/translations.html', u'analytics.html', u'archives.html', u'article.html', u'article_infos.html', u'author.html', u'authors.html', u'base.html', u'categories.html', u'category.html', u'comments.html', u'disqus_script.html', u'github.html', u'gosquared.html', u'index.html', u'page.html', u'pagination.html', u'period_archives.html', u'piwik.html', u'tag.html', u'taglist.html', u'tags.html', u'translations.html', u'twitter.html']
DEBUG: Cannot load cache /home/matthew/Projects/blogbug/cache/PagesGenerator-Readers (this is normal on first run). Proceeding with empty cache.
  | [Errno 2] No such file or directory: u'/home/matthew/Projects/blogbug/cache/PagesGenerator-Readers'
DEBUG: Read file 2015-05-01_test2.ipynb -> Article
DEBUG: Signal article_generator_preread.send(ArticlesGenerator)
DEBUG: Successfuly imported extension module "markdown.extensions.codehilite".
DEBUG: Successfully loaded extension "markdown.extensions.codehilite.CodeHiliteExtension".
DEBUG: Successfuly imported extension module "markdown.extensions.extra".
DEBUG: Successfuly imported extension module "markdown.extensions.smart_strong".
DEBUG: Successfully loaded extension "markdown.extensions.smart_strong.SmartEmphasisExtension".
DEBUG: Successfuly imported extension module "markdown.extensions.fenced_code".
DEBUG: Successfully loaded extension "markdown.extensions.fenced_code.FencedCodeExtension".
DEBUG: Successfuly imported extension module "markdown.extensions.footnotes".
DEBUG: Successfully loaded extension "markdown.extensions.footnotes.FootnoteExtension".
DEBUG: Successfuly imported extension module "markdown.extensions.attr_list".
DEBUG: Successfully loaded extension "markdown.extensions.attr_list.AttrListExtension".
DEBUG: Successfuly imported extension module "markdown.extensions.def_list".
DEBUG: Successfully loaded extension "markdown.extensions.def_list.DefListExtension".
DEBUG: Successfuly imported extension module "markdown.extensions.tables".
DEBUG: Successfully loaded extension "markdown.extensions.tables.TableExtension".
DEBUG: Successfuly imported extension module "markdown.extensions.abbr".
DEBUG: Successfully loaded extension "markdown.extensions.abbr.AbbrExtension".
DEBUG: Successfully loaded extension "markdown.extensions.extra.ExtraExtension".
DEBUG: Successfuly imported extension module "markdown.extensions.meta".
DEBUG: Successfully loaded extension "markdown.extensions.meta.MetaExtension".
DEBUG: Applying preprocessor: coalesce_streams
DEBUG: Applying preprocessor: CSSHTMLHeaderPreprocessor
DEBUG: Applying preprocessor: HighlightMagicsPreprocessor
DEBUG: Attempting to load template basic.tpl
-> Loaded template basic.tpl
DEBUG: Signal article_generator_context.send(ArticlesGenerator, <metadata>)
-> Writing /home/matthew/Projects/blogbug/output/terst2.html
-> Writing /home/matthew/Projects/blogbug/output/index.html
-> Writing /home/matthew/Projects/blogbug/output/tags.html
-> Writing /home/matthew/Projects/blogbug/output/categories.html
-> Writing /home/matthew/Projects/blogbug/output/authors.html
-> Writing /home/matthew/Projects/blogbug/output/archives.html
-> Writing /home/matthew/Projects/blogbug/output/category/misc.html
-> Writing /home/matthew/Projects/blogbug/output/author/matthew.html
Done: Processed 1 article(s), 0 draft(s) and 0 page(s) in 0.28 seconds.
```

**Comment by MatthewGilbert** [comment]
Fixed in fdcd095a0719e786d21dd0a0231793ab5cb1d8c3
